pyproject.toml

```diff
@@ -7,7 +7,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "uid2_client"
-version = "2.0.3"
+version = "2.1.0"
```

Comment:
should this be 2.2.0? given the current version is 2.1.0 already? https://pypi.org/project/uid2-client/

Reply:
bumped the version to 2.2.0